libkbfs/tlf_journal.go

```diff
@@ -1218,26 +1215,14 @@ func (j *tlfJournal) flushOneMDOp(
 
 	mdServer := j.config.MDServer()
 
-	// TODO: Do we need `end` at all, or can we just pass
-	// `maxMDRevToFlush+1` here?  The only argument for `end` is that
-	// it might help if the block and MD journals are out of sync.
```

Comment:
Even if the journals are out of sync, the removed code below is effectively equivalent to passing in maxMDRevToFlush+1 anyway.